.github/workflows/nodejs.yml

```
@@ -9,27 +9,20 @@ on:
     branches:
       - '**'
 
-env:
-  CI: true
-
 jobs:
   lint:
-    name: Lint on Node.js ${{ matrix.node }} and ${{ matrix.os }}
-
-    runs-on: ${{ matrix.os }}
+    name: Lint on Node.js LTS
 
-    strategy:
-      matrix:
-        node: [12]
-        os: [ubuntu-latest]
+    runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
+      - name: Use LTS Node.js
```

Let's use the same naming as `lint.name`!

```suggestion
      - name: Use Node.js LTS
```

---

Good point. Keeping consistency would be great.